src/django/api/exceptions.py

```diff
@@ -1,13 +1,19 @@
 from rest_framework.exceptions import APIException
 
 
-class ForbiddenException(APIException):
-    status_code = 403
-    default_detail = 'You are not allowed to perform this action.'
-    default_code = 'forbidden'
-
-
 class BadRequestException(APIException):
     status_code = 400
     default_detail = 'There was a problem with your request.'
     default_code = 'bad_request'
+
+
+class NotAuthorized(APIException):
```

Comment:
Consider using the built-in [NotAuthenticated](https://www.django-rest-framework.org/api-guide/exceptions/#notauthenticated)

Reply:
b25cbb36debb900721771677409dc8c6e1659fae